apps/web/vibes/soul/primitives/navigation/index.tsx

```diff
@@ -98,6 +98,7 @@ interface Props<S extends SearchResult> {
   cartLabel?: string;
   accountLabel?: string;
   searchLabel?: string;
+  toggleButtonLabel?: string;
```

Comment:
Suggestion on naming: `mobileMenuTriggerLabel`

Reply:
Yeah, this works for me.